platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/paos/PaosInInterceptor.java

```diff
@@ -276,6 +280,14 @@ public void handleMessage(Message message) throws Fault {
     }
   }
 
+  private String getStringValue(Object val) {
```

Comment:
❓ Why not just always do `String.valueOf`?

Reply:
Don't want to string value of if its already a string

Reply:
Theoretically valueOf(String) resolves to valueOf(Object) which just calls `obj.toString()`, so not really destructive.. so i'm saying it would also work and be less code... but i am so lazy andrew why do you make my guilty conscious run amok

Reply:
I updated, you evil and cruel person.  😭